src/composables/transactionTx.ts

```diff
@@ -115,7 +115,8 @@ export function useTransactionTx({
       case TRANSACTION_OWNERSHIP_STATUS.current:
         return {
           ...activeAccount.value,
-          label: i18n.t('transaction.overview.accountAddress'),
+          // @ts-ignore
+          label: i18n.global.t('transaction.overview.accountAddress'),
```

Comment:
Why are we using `i18n.global.t` and in some places use `useI18n`?

Reply:
`useI18n`  "_Must be called at the top of a `setup` function_" so in some cases we cannot use it and need to fallback to `i18n.global.t`